awsconfigs/apps/pipeline-static/s3/config

```diff
@@ -6,15 +6,10 @@ artifactRepository:
         keyPrefix: artifacts,
         endpoint: $(kfp-artifact-storage-endpoint),
         insecure: false,
-        accessKeySecret: {
```

Comment:
why is this part removed from pipeline-static ? These are used for static-credentials, has something changed?

Reply:
it was a mistake, just fixed.